[2024-05-14T07:05:10.118+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T07:05:10.129+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T07:05:05.040069+00:00 [queued]>
[2024-05-14T07:05:10.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T07:05:05.040069+00:00 [queued]>
[2024-05-14T07:05:10.134+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T07:05:10.141+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): data_crawler_detail> on 2024-05-14 07:05:05.040069+00:00
[2024-05-14T07:05:10.145+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=116) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T07:05:10.146+0000] {standard_task_runner.py:63} INFO - Started process 118 to run task
[2024-05-14T07:05:10.146+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'data_crawler_detail', 'manual__2024-05-14T07:05:05.040069+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/data_pipeline_jobs104-checkpoint.py', '--cfg-path', '/tmp/tmpmaeodz7a']
[2024-05-14T07:05:10.149+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask data_crawler_detail
[2024-05-14T07:05:10.175+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.data_crawler_detail manual__2024-05-14T07:05:05.040069+00:00 [running]> on host 9bf348f884a5
[2024-05-14T07:05:10.223+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='data_crawler_detail' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T07:05:05.040069+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T07:05:05.040069+00:00'
[2024-05-14T07:05:10.225+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T07:05:10.232+0000] {logging_mixin.py:188} INFO - data_crawler_detail !!!
[2024-05-14T07:05:10.233+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T07:05:10.233+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T07:05:10.239+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=data_crawler_detail, run_id=manual__2024-05-14T07:05:05.040069+00:00, execution_date=20240514T070505, start_date=20240514T070510, end_date=20240514T070510
[2024-05-14T07:05:10.291+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T07:05:10.308+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T07:05:10.310+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
